grow-website-gatsby/src/pages/components/Workshop/Workshop.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment } from "react"
+import Moment from "moment"
+import AnchorLink from "react-anchor-link-smooth-scroll"
+import { Event } from "schema-dts"
+import { JsonLd } from "react-schemaorg"
+
+import * as styles from "../../Index.module.scss"
+import WorkshopData from "./WorkshopData"
+
+const Workshop = () => {
+  return (
+    <div id="workshop">
+      <div className={styles.sectionWorkshop}>
+        <div className={styles.headingLarge}>Workshops</div>
+        <div className={styles.headingXSmall}>Upcoming Event</div>
+        <div className={styles.workshopImageContainer}>
+          <div className={styles.ratioContent}>
+            <img
+              className={styles.workshopImage}
+              src={WorkshopData.image}
+              alt={WorkshopData.imageAltText}
+            />
+          </div>
+        </div>
+        <div className={styles.headingMedium}>{WorkshopData.name}</div>
+        <div className={styles.uWidthSixtyPercent}>
+          <div className={styles.headingSubtitle}>
+            {Moment(WorkshopData.dateTime).format("ha, dddd do MMMM YYYY")} |
```

Comment:
We could look into using `date-fns` as opposed to `Moment`, as it has modular imports and all the perks of tree shaking :) . Thoughts?

Reply:
Yep we could look into that - for now I might merge this, but could look into it for a new PR :)